### PR TITLE
Update coinflip.js

### PR DESCRIPTION
**Pull request by umbraNewt**
Added optional coin amount to flip.
Users will be able to choose how many coins to flip. The bot will print how many of each side was flipped.

### CODE REVIEW

**Comment by Kashalls** [comment]
> /home/travis/build/dirigeants/klasa-pieces/commands/Fun/coinflip.js
  14:29  error  Irregular whitespace not allowed                                       no-irregular-whitespace
  21:6   error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  29:1   error  Block must be padded by blank lines                                    padded-blocks
✖ 3 problems (3 errors, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.

If you fix these three issues, it should pass. Instead of msg.channel.send(), use return msg.send(). Otherwise I see no issues other than mentioned eslint.

**Comment by Kashalls** [comment]
Still didn´t pass eslint.

**Comment by umbraNewt** [comment]
thanks for the help Kashall, I appreciate it

File: commands/Fun/coinflip.js [hunk 1]
```diff
@@ -6,12 +6,25 @@ module.exports = class extends Command {
...
-			description: 'Flips a (pseudo) fair coin.'
+			description: 'Flips one or more coins',
+			usage: '[coins:int]'
 		});
 	}
 
-	async run(msg) {
-		return msg.reply(`You flipped ${Math.random() > 0.5 ? 'Heads' : 'Tails'}.`);
+	async run(msg, [coins = 0]) {
+		var heads = 0;
```
Comment (kyranet) [inline commands/Fun/coinflip.js]: Should use `let`.
https://eslint.org/docs/rules/no-var

File: commands/Fun/coinflip.js [hunk 2]
```diff
@@ -6,12 +6,25 @@ module.exports = class extends Command {
...
+		var tails = 0;
+
+		switch (coins > 1) {
```
Comment (kyranet) [inline commands/Fun/coinflip.js]: Make this a `if`, `switch` for this case are huge and useless.

File: commands/Fun/coinflip.js [hunk 3]
```diff
@@ -6,12 +6,25 @@ module.exports = class extends Command {
...
+			case true:
+				for (var i = 0; i < coins; i++) {
+					if (Math.random() > 0.5) heads++;
+					else tails++;
+				}
+				return msg.channel.send(`You flipped ${coins} coins. ${heads} ${heads === '1' ? 'was' : 'were'} heads, and ${tails} ${tails === '1' ? 'was' : 'were'} tails.`);
```
Comment (kyranet) [inline commands/Fun/coinflip.js]: Use `msg.send` instead, to enable the editableCommands feature from Klasa.

File: commands/Fun/coinflip.js [hunk 4]
```diff
@@ -6,12 +6,24 @@ module.exports = class extends Command {
...
+	async run(msg, [coins = 0]) {
+		let heads = 0;
+		let tails = 0;
+
+		if (coins > 1) {
+			for (let i = 0; i < coins; i++) {
+				if (Math.random() > 0.5) heads++;
+				else tails++;
+			}
+			msg.send(`You flipped ${coins} coins. ${heads} ${heads === '1' ? 'was' : 'were'} heads, and ${tails} ${tails === '1' ? 'was' : 'were'} tails.`);
```
Comment (kyranet) [inline commands/Fun/coinflip.js]: Please, return the messages sent.

File: commands/Fun/coinflip.js [hunk 5]
```diff
@@ -6,12 +6,24 @@ module.exports = class extends Command {
...
 
-			description: 'Flips a (pseudo) fair coin.'
+			description: 'Flips one or more coins',
+			usage: '[coins:int]'
 		});
 	}
 
-	async run(msg) {
-		return msg.reply(`You flipped ${Math.random() > 0.5 ? 'Heads' : 'Tails'}.`);
+	async run(msg, [coins = 0]) {
```
Comment (kyranet) [inline commands/Fun/coinflip.js]: You do not need `async` here. (Please return messages sent).

File: commands/Fun/coinflip.js [hunk 6]
```diff
@@ -6,12 +6,24 @@ module.exports = class extends Command {
...
+		let tails = 0;
+
+		if (coins > 1) {
+			for (let i = 0; i < coins; i++) {
+				if (Math.random() > 0.5) heads++;
+				else tails++;
+			}
+			msg.send(`You flipped ${coins} coins. ${heads} ${heads === '1' ? 'was' : 'were'} heads, and ${tails} ${tails === '1' ? 'was' : 'were'} tails.`);
+		} else {
+			msg.send(`You flipped ${Math.random() > 0.5 ? 'Heads' : 'Tails'}.`);
```
Comment (kyranet) [inline commands/Fun/coinflip.js]: Same here.

**Pull request merged**